content/doc/book/managing/user-content.adoc

```diff
@@ -0,0 +1,25 @@
+---
+layout: section
+title: User Content
+---
+ifdef::backend-html5[]
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:toclevels: 4
+:hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+endif::[]
+
+= User Content
+
+Jenkins has a mechanism known as "User Content", where administrators can place files inside  `$JENKINS_HOME/userContent`,
+and these files are served from link:http://yourhost/jenkins/userContent[http://yourhost/jenkins/userContent]. This can be thought of as a mini HTTP server to serve
```

Comment:
Thanks for the pull request.  Since this is an example URL, it is better to have it displayed as the text of the URL without actually being a hyperlink.  When we make it a hyperlink, some readers attempt to click it and are directed away from the site to a non-existent page.
```suggestion
and these files are served from \http://yourhost/jenkins/userContent. This can be thought of as a mini HTTP server to serve
```